<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogInWindow</class>
 <widget class="QDialog" name="LogInWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>261</width>
     <height>207</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="main_VerticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="login_verticalLayout">
        <item>
         <widget class="QLabel" name="login_label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Логин:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="login_lineEdit">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="password_verticalLayout">
        <item>
         <widget class="QLabel" name="password_label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Пароль:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="password_lineEdit">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="error_lbl">
        <property name="styleSheet">
         <string notr="true">color: red;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="signUp_btn">
        <property name="locale">
         <locale language="Prussian" country="World"/>
        </property>
        <property name="text">
         <string>Регистрация</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttons_horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="login_btn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Войти</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel_btn">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
